src/pages/NotFound.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import gif from '../assets/wtfcat.gif';
+
+const NotFoundPage = () => {
+  const containerStyle = {
+    backgroundImage: `url(${gif})`,
+    backgroundSize: 'cover',
+    backgroundColor: '#0000FF',
+    height: '100vh',
+    display: 'flex',
+    justifyContent: 'center',
+    alignItems: 'center',
+    textAlign: 'center',
+    color: 'white',
+    fontSize: '2rem',
+  };
```

Comment:
Setting up styles on a component is definitely an approach that some developers take and am curious what brought you to going this approach versus using your CSS file and calling on classnames?